docs/design/datacontracts/CodeVersions.md

```diff
@@ -40,6 +40,9 @@ public virtual bool CodeVersionManagerSupportsMethod(TargetPointer methodDesc);
 
 // Return the instruction pointer corresponding to the start of the given native code version
 public virtual TargetCodePointer GetNativeCode(NativeCodeVersionHandle codeVersionHandle);
+
+// Gets the GCStressCodeCopy pointer if available, otherwise returns TargetPointer.Null
+public virtual TargetPointer GetGCStressCodeCopy(NativeCodeVersionHandle codeVersionHandle);
```

Comment:
`GCCoverageInfo` is a runtime type which holds state about the functions GCCoverage, including a copy of the original code before any GC points were inserted. In the runtime this field is `savedCode`.

The DAC type `DacpMethodDescData` has a field which should be populated with a pointer to the `gcCoverageInfo.savedCode`, `GCStressCodeCopy`. 

My preference is to use `GetGCStressCodeCopy` to mimic the DAC type and because it better describes what is actually being returned.